_rules/SC2-4-6-descriptive-headings.md

```diff
@@ -21,33 +22,34 @@ authors:
 
 ### Applicability
 
-The rule applies to any element with the the [semantic role](#semantic-role) of `heading`.
+This rule applies to any element with the [semantic role](#semantic-role) of heading that is either visible or included in the accessibility tree.
```

Comment:
Remember link to definitions: 
[visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree) = `[visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree)`

Reply:
Done :)